### PR TITLE
feat(authent): prompt Registry password from Stdin if not provided

### PR DESCRIPTION
**Pull request by educhastenier**
+ Move printFinalNote() call only if build successful

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -321,10 +323,20 @@ func buildCustomDockerImage(ctx context.Context, dockerClient *client.Client, do
 			"BONITA_BASE_IMAGE": baseImage},
 	}
 	if *verbose {
-		fmt.Println("Using base docker image:", baseImage)
 		fmt.Println("Building new image:", *tag)
 	}
 
+	if *registryUsername != "" && *registryPassword == "" {
+		fmt.Printf("Enter your password to access Bonita Artifact Repository corresponding to '%v':", *registryUsername)
```
Comment (akantcheff) [inline main.go:330]: The registry might not be our Bonita Artifact Repository (eg a private registry with a mirror configuration to the Artifactory)

Comment (educhastenier) [reply main.go:330]: Ok, I'll change this.

Comment (educhastenier) [reply main.go:330]: > Enter your password to access the Docker Registry corresponding to '%v':

Is it ok for you?

**Comment by educhastenier** [comment]
@akantcheff I just added a new commit to complete usage with this contribution:
[`c717ae3` (#26)](https://github.com/bonitasoft/bonita-application-packager/pull/26/commits/c717ae348632dca5e491059c508752ec101a0df9)

**Pull request merged**